src/main/java/seedu/address/logic/commands/sortstudentcommands/SortStudentParameter.java

```diff
@@ -0,0 +1,8 @@
+package seedu.address.logic.commands.sortstudentcommands;
+
+/**
+ * Possible parameters for SortStudentCommand
+ */
+public enum SortStudentParameter {
+    NAME, STUDENTID, EMAIL, EMPTY, INVALID
+}
```

Comment:
Since the different parameters are known beforehand, using enums could be much safer from a development standpoint (especially when dealing with case sensitivity with strings). There is also not be a significant efficiency difference between them thus, I chose to refactor using enums.